### PR TITLE
Fix/volunteers data

### PR DESCRIPTION
**Pull request by AnandDhakane01**

### CODE REVIEW

File: lib/services/volunteer.js [hunk 1]
```diff
@@ -19,8 +19,12 @@ module.exports = class UserRoleService extends Schmervice.Service {
...
         volunteers.map((volunteer) => {
           return Classes.query()
-            .where('facilitator_id', volunteer.id)
-            .orWhere('facilitator_email', volunteer.email)
+            .where((builder) =>
+              builder
+                .where('facilitator_id', volunteer.id)
+                .orWhere('facilitator_email', volunteer.email)
+            )
+            .where('start_time', '<', 'NOW()')
```
Comment (pysaquib) [inline lib/services/volunteer.js:27]: `NOW( )` might tell you the time in GMT. We have class start_time stored in prod DB used as IST. You should convert the time into IST before comparing. 
Attaching screenshot of time from prod DB.
Screenshot_select-area_20211210155446

File: lib/services/pathway.js [hunk 2]
```diff
@@ -6,6 +6,49 @@ const { transaction } = require('objection');
 const { errorHandler } = require('../errorHandling');
 
 module.exports = class PathwayService extends Schmervice.Service {
+  async addPathways(userId, pathwayIds, txn) {
```
Comment (pysaquib) [inline lib/services/pathway.js:9]: You moved these functions into Pathway service but I don't see changes where they are used. They are probably still used like this `userService.addPathways( )`

File: lib/services/pathway.js [hunk 3]
```diff
@@ -6,6 +6,49 @@ const { transaction } = require('objection');
...
+    const pathways = pathwayIds.map((id) => ({ user_id: userId, pathway_id: id }));
+    try {
+      await StudentPathways.query(txn).insert(pathways);
+      return [null, userId];
+    } catch (err) {
+      return [errorHandler(err), null];
+    }
+  }
+
+  async removePathways(userId, pathwayIds, txn) {
```
Comment (pysaquib) [inline lib/services/pathway.js:27]: https://github.com/navgurukul/sansaar/pull/454/files#r766564139

Same comment

**Review by pysaquib — CHANGES_REQUESTED** [review]
Reviewed

File: lib/services/volunteer.js (hunk 1 above)
Comment (AnandDhakane01) [reply lib/services/volunteer.js:27]: read about it and it seems ` NOW() ` returns the current date and time based on the database server’s time zone setting.

**Pull request merged**